platform.json

```diff
@@ -17,7 +17,8 @@
     "https://dl.bintray.com/platformio/dl-packages/manifest.json",
     "https://sourceforge.net/projects/platformio-storage/files/packages/manifest.json/download",
     "http://dl.platformio.org/packages/manifest.json",
-    "https://raw.githubusercontent.com/eerimoq/simba/master/make/platformio/manifest.json"
+    "https://raw.githubusercontent.com/eerimoq/simba/master/make/platformio/manifest.json",
+    "https://raw.githubusercontent.com/eerimoq/pumbaa/master/make/platformio/manifest.json"
```

Comment:
Go ahead and release with two manifests. I'll likely not merge the two manifests to one unless it simplifies something for you in PlatformIO.